neuro_san/internals/run_context/langchain/core/langchain_run_context.py

```diff
@@ -197,7 +197,7 @@ async def create_resources(self, agent_name: str,
                     else:
                         self.tools.append(tool)
 
-        prompt_template: ChatPromptTemplate = await self._create_prompt_template(instructions, assignments)
+        prompt_template: ChatPromptTemplate = await self._create_prompt_template(instructions)
```

Comment:
I believe this was the only reason to pass assignments in the create_resource() call in the first place,
which leads me to the comment about eventually removing it from the create_resources() method altogether in later PR (see comment above)

Reply:
Hmm. Might not be possible with the openai_run_context.  Cross that bridge when we get there.